CONTRIBUTING.md

```diff
@@ -171,6 +171,10 @@ The list of labels for current issues can be found [here][turing-way-labels] and
 
 - [![outreach](https://img.shields.io/badge/-Outreach-fcbae8.svg)][labels-outreach] _These issues relate to topics to reach out to the community._
 
+- [![good-first-PR-review](https://img.shields.io/badge/-good%20first%20PR%20review-C992E0.svg)](https://github.com/alan-turing-institute/the-turing-way/labels/good%20first%20PR%20reviews) _These pull requests are for the new maintainers of The Turing Way to start with reviewing and merging tasks_
```

Comment:
```suggestion
- [![good-first-PR-review](https://img.shields.io/badge/-good%20first%20PR%20review-C992E0.svg)] These pull requests are for the new members of The Turing Way community who want to start with reviewing and approving some simple pull requests._
```

Reply:
Does clicking on the label redirect to [this](https://github.com/alan-turing-institute/the-turing-way/labels/good%20first%20PR%20review) link? I tried to refer to the format as given for labels but couldn't implement the redirection part, hence added the link.

Reply:
That format works for other labels, this should work as well.

Reply:
Oh, and in suggestions made with a comment this the original comment on this thread, you can directly add then by clicking on "commit suggestion" so you don't need to re-edit them in.

Reply:
Yes, did that for the other change. Will do that here as well.